### DISCUSSION TITLE
It's not working with `translate()`

### DISCUSSION THREAD

**Discussion by sdkakcy** [spatie/pest-expectations#7] [Ideas]
If rule contains translate() with fail closure this is not working. Because the given $fail closure cannot be chain with translate().

<img width="661" alt="Screenshot 2023-04-24 at 23 50 39" src="https://user-images.githubusercontent.com/32255629/234114074-5f40814d-b9bb-4fb7-8639-584176527fe0.png">

<img width="673" alt="Screenshot 2023-04-24 at 23 52 15" src="https://user-images.githubusercontent.com/32255629/234114391-f42c9ed4-e9be-4c18-9f0a-cd387d75a575.png">

Is there a way to solve it?

**Comment by freekmurze** [comment]
This is currently unsupported, if you find a clean way of supporting this, I'd accept a PR. Make sure to include tests
